core/sections/clause_8_vector.adoc

```diff
@@ -7,6 +7,7 @@
 * fill:     Fill [0..1]
 * stroke:   Stroke [0..1]
 * marker:   Marker [0..1]
+* label:    Label [0..1]
```

Comment:
label should not be here. It is added in 10-labeling Labeling conformance class.

Reply:
removed label property